pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,12 @@
-function omf.install_package
+# Return success for at least one successful install.
+function omf.install_package 
+  set -l return_code $OMF_UNKNOWN_ERR
   for search in $argv
     omf.install --pkg $search
+    set -l install_code $status
+    if test $install_code -eq 0
```

Comment:
I still think this is unexpected.

I have worked on a bigger change for the install command but didn't have the chance to complete it. You might want to use it as a starting point?

https://github.com/oh-my-fish/oh-my-fish/commit/26c3b722711cc98688e63c6b39b267f5f676a235